.gitignore

```diff
@@ -25,10 +25,10 @@ dist
 jsconfig.json
 target
 guui.zip
-packages/design/gatsby/public
```

Comment:
i had to move this to `/design` for yarn's submodule system to not freak out

Reply:
I prefer it anyway! 👍